lib/process-services-cloud/src/lib/task/task-form/components/task-form-cloud.component.ts

```diff
@@ -126,6 +129,15 @@ export class TaskFormCloudComponent implements OnChanges {
             this.taskDetails = details;
             this.loading = false;
         });
+
+        this.taskCloudService.getCandidateUsers(this.appName, this.taskId).subscribe((users: Array<string>) => {
+            this.candidateUsers = users;
+        });
+
+        this.taskCloudService.getCandidateGroups(this.appName, this.taskId).subscribe((groups: Array<string>) => {
```

Comment:
Same as above, guess error handling is missing, or you should return empty result from the api